hack/make/verify-integration-tests

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
nit: What do you think about calling this "build-integration-test-binary" instead?